src/Parser.js

```diff
@@ -680,7 +680,7 @@ export default class Parser {
         if (res == null) {
             return null;
         }
-        const match = (/^(#[a-f0-9]{3}|#?[a-f0-9]{6}|[a-z]+)$/i).exec(res.text);
+        const match = (/^(#[a-f0-9]{3}|#?[a-f0-9]{6}|#?[a-f0-9]{8}|[a-z]+)$/i).exec(res.text);
```

Comment:
There's also 4-digit hex codes. See https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color

Is there some way to avoid the repetition here? I'm not sure.